warehouse/templates/packaging/detail.html

```diff
@@ -80,7 +80,16 @@ <h3 class="title"><a href="{{ request.route_path('packaging.project', name=relea
     <p class="help-link"><a href="#" {{ l20n("howToInstall") }}>How do I install this?</a></p>
   </div>
 </section>
-
+{% if release.insecure %}
+<section class="horizontal-section -bad -medium">
```

Comment:
Can we try a 'thin' style here instead?  It looks a little fat to me - I think the red is distinct enough without taking up the extra screen real estate.